src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CaseActionsForCaseWorkerController.java

```diff
@@ -111,6 +113,9 @@ public class CaseActionsForCaseWorkerController {
     private final CaseManagementLocationService caseManagementLocationService;
 
     private final NocRespondentHelper nocRespondentHelper;
+    @Value("${ccd_gateway_base_url}")
```

Comment:
does it make more sense to pass this into the service rather than controlller

Reply:
It was in the service. This way it is easier to test. I look into it.